.github/workflows/workflow.yml

```diff
@@ -0,0 +1,90 @@
+name: Main workflow
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      fail-fast: false
+      matrix:
+        operating-system: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Setup Node.js 12
+      uses: actions/setup-node@v1
```

Comment:
Let's revert this file before merging since I have already added it in the separate PR

Reply:
Done. Thank you.